src/api/app/services/github_status_reporter.rb

```diff
@@ -0,0 +1,44 @@
+class GithubStatusReporter < SCMStatusReporter
+  def call
+    github_client = Octokit::Client.new(access_token: @scm_token,
+                                        api_endpoint: @event_subscription_payload[:api_endpoint])
+    # https://docs.github.com/en/rest/reference/repos#create-a-commit-status
+    github_client.create_status(@event_subscription_payload[:target_repository_full_name],
+                                @event_subscription_payload[:commit_sha],
+                                @state,
+                                status_options)
+    if @workflow_run.present?
+      @workflow_run.save_scm_report_success(request_context)
+      RabbitmqBus.send_to_bus('metrics', "scm_status_report,status=success,scm=#{@event_subscription_payload[:scm]} value=1")
+    end
+  rescue Octokit::InvalidRepository => e
+    package = Package.find_by_project_and_name(@event_payload[:project], @event_payload[:package])
+    return if package.blank?
+
+    tokens = Token::Workflow.where(scm_token: @scm_token).pluck(:id)
+    return if tokens.none?
+
+    EventSubscription.where(channel: 'scm', token: tokens, package: package).delete_all
+
+    @workflow_run.save_scm_report_failure("Failed to report back to GitHub: #{e.message}", request_context) if @workflow_run.present?
+  rescue Octokit::Error => e
+    rescue_with_handler(e) || raise(e)
+  rescue Faraday::ConnectionFailed => e
+    @workflow_run.update_as_failed("Failed to report back to GitHub: #{e.message}")
```

Comment:
This line changed in a previous PR. See [this commit](https://github.com/openSUSE/open-build-service/pull/12786/commits/fb6f6f34ee3230f9e00e7a2c267d9b99d91f3b25)

The specs need to be adapted as well.